res/controllers/Traktor Kontrol S4 MK2.hid.xml

```diff
@@ -9,7 +9,7 @@
             <product protocol="hid" vendor_id="0x17cc" product_id="0x1310" usage_page="0xff01" usage="0x1" interface_number="0x4" />
         </devices>
     </info>
-    <controller id="Traktor">
+    <controller id="KontrolS4">
```

Comment:
Why was this changed in this PR?

Reply:
Because I looked into the XML syntax used across other controllers for setting the id and then stumbled upon this inconsistency ^^

Reply:
Okay, but this has nothing to do with the Z1. In the future when you make miscellaneous changes like this, please do them in their own commit instead of including them with unrelated changes.